docs/index.rst

```diff
@@ -283,6 +316,47 @@ With the above report, Betelgeuse performs the following:
     to run ``test-command`` command before ``test-run`` to make sure all
     required test cases are created/updated accordingly.
 
+The test-run command allows setting custom fields in order to better define the
+environment. There are two way to define custom fields the ``key=value`` and
+JSON formats. The former is a shortcut when you want to define plain strings as
+the value of a custom field, on the other hand, the later is better suited when
+the type of the custom field matters, for example, if a custom field expects a
+boolean as a value.
+
+Below is an example using the ``key=value`` format:
+
```

Comment:
How about:

The test-run command allows setting custom fields in order to better define the environment. There are two ways to define custom fields:

``key=value`` format
    This a shortcut when you want to define plain strings as the value of   a custom field.
JSON format
    This approach suits better when the type of the custom field matters.  For example, if a custom field expects a boolean as a value.

Example for ``key=value`` format: